lib/secretariat/invoice.rb

```diff
@@ -17,11 +17,17 @@
 require 'bigdecimal'
 
 module Secretariat
+  using ObjectExtensions
+
   Invoice = Struct.new("Invoice",
     :id,
     :issue_date,
     :service_period_start,
     :service_period_end,
+    :delivery_date,
+    :invoice_type,
+    :invoice_reference_id,
```

Comment:
```suggestion
    :invoice_reference_number,
```

I'd call this number because `id` probably relates to database ids for many developers.

Reply:
Updated on other PR https://github.com/fortytools/ruby-secretariat/pull/10